Gaijinizer patch/Map216.txt

```diff
@@ -454,30 +474,30 @@ stopped". It doesn't really stop time. .#MTLed
 そもそも『時間停止』なぞ、規模が尋常ではない。
 その世界の全ての次元が範囲じゃぞ？
 > CONTEXT: Map216/events/5/pages/1/168/Dialogue/enemy17
-Rather...that twins was out of spec.
-In the first place, the scale is not unusual as
-it is "time stop". All the dimensions of the
-world are in range?#MTLed
+But the Twins are quite extraordinary.
+Stopping time is a rather uncommon ability, and
+that ability includes all of this world's
+dimensions.#RonIffyWording
 > END STRING
```

Comment:
I think this is what they're getting at:
>_**Stopping time is a rather rare ability, as
it would have to cover the entire world, including
all extra dimensional spaces.**_